benchmarks/multimodal/sweep/server.py

```
@@ -0,0 +1,126 @@
+        return self._process is not None and self._process.poll() is None
+
+    def start(
+        self,
+        workflow_script: str,
+        model: str,
+        extra_args: Optional[List[str]] = None,
+        env_overrides: Optional[dict] = None,
+    ) -> None:
+        """Launch the workflow script and block until the model is served."""
+        if self.is_running:
+            raise RuntimeError("Server is already running. Call stop() first.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)